Command/GenKeysCommand.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Lexik\Bundle\JWTAuthenticationBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * GenKeysCommand.
+ *
+ * @author zorn-v
+ */
+class GenKeysCommand extends Command
+{
+    /**
+     * @var string
+     */
+    private $privateKey;
+
+    /**
+     * @var string
+     */
+    private $publicKey;
+
+    /**
+     * @var string
+     */
+    private $passphrase;
+
+    /**
+     * Constructor.
+     *
+     * @param string $privateKey
+     * @param string $publicKey
+     * @param string $passphrase
+     */
+    public function __construct($privateKey, $publicKey, $passphrase)
+    {
+        $this->privateKey = $privateKey;
+        $this->publicKey  = $publicKey;
+        $this->passphrase = $passphrase;
+
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setName('lexik:jwt:gen-keys')
+            ->setDescription('Generate JWT keys')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $fs = new Filesystem();
+        $fs->mkdir(dirname($this->privateKey));
+        $fs->mkdir(dirname($this->publicKey));
```

Comment:
The bundle now supports raw keys instead of file paths (think env var), it would be great to support writing the generated key to stdout with an option

Reply:
I don't know now how to better inject `private_key_path` and `public_key_path` as they deprecated but new `secret_key` and `public_key` not in container (as deprecated ones)